tests/testthat/test-dependencies.R

```diff
@@ -62,11 +65,13 @@ test_that("dependency_table works", {
         repo = paste0("openpharma/", package_name),
         host = rep("https://github.com", 6),
         cache_dir = unlist(Map(get_repo_cache_dir, repo, host, local = grepl("^local ", ref))),
-        sha =  rep("test", 6),
+        sha = rep("test", 6),
         accessible = rep(TRUE, 6),
         installable = rep(TRUE, 6)
-      ) %>% dplyr::select(package_name, type, distance, ref, repo, host, sha, cache_dir, accessible, installable, install_index)
-
+      ) %>%
+        dplyr::select(
+          c(package_name, type, distance, ref, repo, host, sha, cache_dir, accessible, installable, install_index)
```

Comment:
convert to strings + dplyr::all_of

Reply:
weridly I tried this locally (in 2 places) and the tests failed - will explore more

Reply:
working on tidyselect 1.2.0 this seems to work ok:

![image](https://user-images.githubusercontent.com/15201933/195300856-682a424a-e9cd-4766-9416-482ab246f7a7.png)